cutils.c

```diff
@@ -590,7 +590,13 @@ size_t utf8_encode_buf16(char *dest, size_t dest_len, const uint16_t *src, size_
  */
 
 /* 2 <= base <= 36 */
-char const digits36[36] = "0123456789abcdefghijklmnopqrstuvwxyz";
```

Comment:
Compiler is right to complain because `sizeof("0123456789abcdefghijklmnopqrstuvwxyz")` is 37, not 36.